### DISCUSSION TITLE
Inspired by the Locker, I'd like to make my own robotic enemy.

### DISCUSSION THREAD

**Discussion by GrumbleGrits** [zealsprince/lc-locker#7] [General]
I've never made anything before, but have been overcome with some amount of motivation and inspiration thanks to this neat creation of yours. it looks, sounds, and behaves incredibly... accidentally running into it around a corner, or scanning it at the end of a long hallway, make for some properly thrilling moments.

I was curious, what tools did you use to create this? more specifically, how did you model it? I'd like to model my own thingy, but have no idea where to start.

**Comment by zealsprince** [comment]
Code was written entirely in Visual Studio since C# is just much nicer to work with in it vs something like just VS Code and then building yourself through some strange custom toolchain. I went back and forth between the implementation in Unity since this was my first enemy and I immediately went with something that is completely different from anything else in the game which didn't make things easier. But hey, good learning experience!

Modeled, rigged and animated in Blender. Textured pretty much from Affinity Photo. I really didn't do any magic process on this one compared to other game projects and scenes I've created in the past. It was all really straight forward. If you'd like, I can post some of my general work flow and setup.

**Reply by GrumbleGrits** [reply]
Yeah, I would love to know more about how you made Locker. I have no experience with anything haha, so could use all the pointers I can get.
